packages/xforms-engine/src/client/BaseItem.ts

```diff
@@ -0,0 +1,7 @@
+import type { TextRange } from './TextRange.ts';
+
+export interface BaseItem {
```

Comment:
I'm wondering if this should be just `Item` now. BaseItem made sense when thinking about Rank and Select extending it, but seeing as there is now hierarchy now does it still make sense?

Reply:
I agree `Item` is a bit vague but so are all the other names I can think of  :/

`BaseItem` is fine as I can't think of anything better!